packages/astro/src/@types/astro.ts

```diff
@@ -494,6 +494,22 @@ export interface AstroUserConfig {
 	 */
 	trailingSlash?: 'always' | 'never' | 'ignore';
 
+	/**
+	 * @docs
+	 * @name scopedStyleStrategy
+	 * @type {('where' | 'class')}
+	 * @default `'where'`
+	 * @description
+	 *
```

Comment:
```suggestion
	 * @version 2.3.5
	 *
```
This might be a good candidate for the "Added In" field?